### PR TITLE
Added localization config, French translation

### PR DESCRIPTION
**Pull request by Gunit2481**
**Pull Request Summary**

-Added translation file for French

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [x] documentation changes

**This pull request makes the following changes:**

**Adds**
- (ex: Add feature A)

**Fixes**
- (ex: Fix validation function)

**Changes**
- (ex: Change document B)

**To-dos**
- Feel free to remove this section if it's not applicable
- Probably add the language code into URL path

- [ ] (ex: add user list)

### CODE REVIEW

File: src/i18n/index.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import enUS from './en-US';
 import zh from './zh';
 import zhTW from './zh-TW';
 import ja from './ja';
+import FR from './FR';
```
Comment (hoonsubin) [inline src/i18n/index.ts:5]: please use lower case

Comment (Gunit2481) [reply src/i18n/index.ts]: Done !

File: src/i18n/index.ts [hunk 2]
```diff
@@ -20,11 +21,16 @@ export const languagesSelector = [
     text: '繁體中文',
     code: 'zh-TW',
   },
+  {
+    text: 'Français',
+    code: 'FR',
```
Comment (hoonsubin) [inline src/i18n/index.ts:26]: Can you also make the code and the script name to use underscore?

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/i18n/index.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import enUS from './en-US';
 import zh from './zh';
 import zhTW from './zh-TW';
 import ja from './ja';
+import fr from './fr';
```
Comment (hoonsubin) [inline src/i18n/index.ts:5]: You can still use `index.ts` for `fr/fr.ts`. The import name should be `fr` (like now)

**Review by hoonsubin — CHANGES_REQUESTED** [review]

**Comment by hoonsubin** [comment]
The UI structure has changed a lot, I'm sorry to ask this, but can you close this PR and translate the page again? I will ensure it is mergable.
Thanks

**Pull request closed without merging**